workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -222,8 +247,11 @@ public void AddGameObjectEntity(Entity entity, GameObject gameObject)
                 case Authority.AuthorityLossImminent:
                     if (!HasComponent<Authoritative<T>>(entity))
                     {
-                        Debug.LogErrorFormat(Errors.IncorrectAuthorityTransition, Authority.AuthorityLossImminent,
-                            Authority.Authoritative);
+                        ErrorDispatcher.HandleError(new InvalidStateTransition<Authority>
+                        {
+                            NewState = Authority.AuthorityLossImminent,
+                            OldState = Authority.NotAuthoritative
```

Comment:
The original arguments were `Authority.AuthorityLossImminent` and `Authority.Authoritative`. Was this an intended change?